inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -628,7 +628,13 @@ V10Parser::GenericLayerParams V10Parser::XmlDeserializer::parseGenericParams(con
         port.precision = type;
         std::vector<std::string> names;
         if (getParameters<std::string>(parentNode, "names", names)) {
-            for (const auto& name : names) {
+            for (size_t i = 0; i < names.size(); i++) {
+                std::string name = names[i];
+                // Restore original name if it contains delimiter
+                while (names[i].at(names[i].length() - 1) == '\\' && i < names.size()) {
+                    name.replace(names[i].length() - 1, 1, ",");
```

Comment:
Because in IR we use `'\,'` combination in order to support names with `','`, but nGraph tensor name shouldn't have `'\'` symbol. in this case I just replace `'\'` to `','` and restore the original name